.travis.yml

```diff
@@ -1,20 +1,32 @@
+sudo: false
+
 language:
   - php
 
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 php:
   - 5.3
   - 5.4
   - 5.5
+  - 7.0
+  - hhvm
 
-env:
-  - SYMFONY_VERSION=2.1.*
-  - SYMFONY_VERSION=2.2.*
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=2.4.*
+matrix:
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: COVERAGE=yes
 
 before_script:
-  - composer require symfony/framework-bundle:${SYMFONY_VERSION}
-  - composer update --dev
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/framework-bundle:${SYMFONY_VERSION}; fi
+  - composer install
+  - if [ "$COVERAGE" != "yes" -a "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
```

Comment:
Loading Xdebug when not needed only slows down tests (HHVM doesn't provide the Xdebug extension).